pkgs/tools/networking/modem-manager/default.nix

```diff
@@ -33,12 +33,9 @@ stdenv.mkDerivation rec {
   doCheck = true;
 
   postInstall = ''
-    # rename to modem-manager to be in style
-    mv $out/etc/systemd/system/ModemManager.service $out/etc/systemd/system/modem-manager.service
-
     # systemd in NixOS doesn't use `systemctl enable`, so we need to establish
     # aliases ourselves.
-    ln -s $out/etc/systemd/system/modem-manager.service \
+    ln --relative -s $out/etc/systemd/system/ModemManager.service \
```

Comment:
This didn't have any effect for me when I tested it in https://github.com/NixOS/nixpkgs/pull/57689

Reply:
@worldofpeace For context, idea was was the `--relative` would fix https://github.com/NixOS/nixpkgs/issues/41055.

I dropped this patch from the PR, we can follow up on that later.

Reply:
Yep, I did similar for that pr trying to fix https://github.com/NixOS/nixpkgs/issues/41055
Probably out of scope for this too.